apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -73,6 +74,19 @@ import {getStore} from '@cdo/apps/redux';
 import {setFailedToGenerateCode} from '@cdo/apps/redux/blockly';
 import {handleCodeGenerationFailure} from './utils';
 import {MetricEvent} from '@cdo/apps/lib/metrics/events';
+import {
+  BlocklyWrapperType,
```

Comment:
I made this `BlocklyWrapperType` to avoid confusion with the variable `BlocklyWrapper` (which did not seem like a good idea to rename). If you have an idea for a better name for the type I am open to it, I couldn't think of anything.

Reply:
I don't have a better idea. :D 
This works for me.